packages/core/strapi/lib/core-api/__tests__/service.test.js

```diff
@@ -146,3 +147,119 @@ describe('Default Service', () => {
     });
   });
 });
+
```

Comment:
to be a bit more consistent now that the service is a folder in core-api can you move those tests in core-api/service/__tests__ and split the index & paginationt tests in 2 files